### DISCUSSION TITLE
Hi, I have a background color question

### DISCUSSION THREAD

**Discussion by cenationwawa** [mkkellogg/GaussianSplats3D#355] [Q&A]
Could you tell me how to change the background color to black? Is there any way?

**Comment by d0rianb** [comment]
You can instanciate your own `THREE.WebGLRenderer`, then set the clear color : `renderer.setClearColor(new THREE.Color(0xeeeeee), 1.0)` and pass it to the viewer when initializing it.

**Comment by cenationwawa** [comment]
Thank you very much. I was thinking of instantiating one if there was no way to expose it.




915159850
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "mkkellogg/GaussianSplats3D"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月25日(星期五) 下午3:55
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [mkkellogg/GaussianSplats3D] Hi, I have a background color question (Discussion #355)





 
You can instanciate your own THREE.WebGLRenderer, then set the clear color : renderer.setClearColor(new THREE.Color(0xeeeeee), 1.0) and pass it to the viewer when initializing it.
